src/polygonId/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const getProofOfAgeRequest = (params: { maxBirthDate: string }) => ({
+  id: 1,
```

Comment:
note that the request needs to be registered just once (and update the query as the day changes - updating is allowed), there's no point in registering a different request for each user as the query registered is user agnostic. What is user-specific is the proof itself. Hence it's fine to hardcode this to `1` (we support just one query that we update as time passes to be that user has at least 21 years)